lib/preprocessor.py

```diff
@@ -206,8 +208,15 @@ def server_start(self, ctype, sname, opts):
             if crash_expected:
                 # disable crash detector
                 self.servers[sname].crash_expected = True
-            self.servers[sname].start(silent=True, rais=True, wait=wait,
-                                      wait_load=wait_load, args=args)
+            is_started = self.servers[sname].start(
+                    silent=True, rais=True, wait=wait,
+                    wait_load=wait_load, args=args,
+                    server_start_timeout=Options().args.server_start_timeout)
```

Comment:
Are we actually need to pass the timeout from here? I would grab it from options somewhere deeper by the call stack: say, from inside TarantoolServer.start(). If we'll need to pass different startup timeouts, we'll implement it later. It is how the test timeout is implemented and I like the simplicity that the approach brings into the code.

Reply:
Fixed